docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,108 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 mapped addresses in `contains`. ({issue}`18497`)
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
+* Improve performance of `GROUP BY`, `DISTINCT`, and other hashing operations. ({issue}`19059`)
+* Fix query failure due to exceeding node memory limits with `INSERT`
+  statements. ({issue}`18771`)
+* Fix incorrectly reporting `ns` instead of `ms` in query plans. ({issue}`19047`)
+
+## Security
+
+* Add access control check to redirected tables when accessing table comments. ({issue}`18514`)
+
+## Delta Lake connector
+
+* Add support for reading tables with deletion vectors. ({issue}`16903`)
+* Add support for writer
```

Comment:
```suggestion
* Add support for Delta Lake writer version 7
```

Or 

```suggestion
* Add support for Delta Lake protocol version 7
```

Reply:
"Delta Lake writer version 7" is better. We don't call it "protocol version".